### ISSUE TITLE
[Feature Request] JUnit like test report in console

### ISSUE THREAD

**Issue by xli** [matusf/openapi-fuzzer#8]
Not sure how we can use current cli in a CI build, if we could have a JUnit like test report in console, it simplifies the integration into a CI build.

**Comment by matusf** [comment]
I agree, now it's not well suited for the CI right now. However, it's planned in a near future. Do you have any suggestions? I was thinking of adding an cli option to run for specified time and then one would be able to access the results from CI via artifacts.

Actually, you can run it even now in the CI. All you need to do is to kill it after some time and add generated report files to artifacts. Not the most elegant way, but it should work for now.

**Comment by xli** [comment]
Something simple like:

For each test, output `.` for success, `E` for error, `F` for failed
Then at the end a summary line about how many success, error and failures.
